client/src/reducers/projects.ts

```diff
@@ -10,48 +10,51 @@ import {
 } from "../actions/projects";
 import { ProjectEventsMap } from "../types";
 
-export const enum Status {
+export enum Status {
   Undefined = 0,
   Loading,
   Loaded,
   Error,
 }
 
-export const enum AppStatus {
-  Accepted,
-  Rejected,
-  Pending,
-  NotFound,
-  Unknown,
+export enum AppStatus {
+  Approved = "APPROVED",
+  Rejected = "REJECTED",
+  Pending = "PENDING",
+  NotFound = "NOT_FOUND",
+  Unknown = "UNKNOWN",
 }
 
-export type ApplicationStatus = {
-  roundID: string;
-  status: AppStatus;
-};
-
 export interface ProjectsState {
   status: Status;
   error: string | undefined;
   ids: number[];
   events: ProjectEventsMap;
-  applications: {
-    [projectId: number]: {
-      status: Status;
-      applicationStatus: [ApplicationStatus];
-    };
-    error: string | undefined;
-  };
+  applications: [
+    {
+      projectId: string;
+      round: {
+        id: string;
+      };
+      status: AppStatus;
+    }
+  ];
 }
 
 const initialState: ProjectsState = {
   status: Status.Undefined,
   error: undefined,
   ids: [],
   events: {},
-  applications: {
-    error: undefined,
-  },
+  applications: [
+    {
+      projectId: "",
```

Comment:
Do we need the `projectId` here, or is it going to be always the same one repeated? Looking at the actions it seems to be the latter?